packages/webapp/pages/account/subscription.tsx

```diff
@@ -0,0 +1,67 @@
+import type { ReactElement } from 'react';
+import React, { useEffect } from 'react';
+import { usePlusSubscription } from '@dailydotdev/shared/src/hooks';
+import type { NextSeoProps } from 'next-seo';
+
+import { useRouter } from 'next/router';
+import { managePlusUrl, plusUrl } from '@dailydotdev/shared/src/lib/constants';
+import { SubscriptionProvider } from '@dailydotdev/shared/src/lib/plus';
+import { isIOSNative } from '@dailydotdev/shared/src/lib/func';
+import { WebKitMessageHandlers } from '@dailydotdev/shared/src/lib/ios';
+import { AccountPageContainer } from '../../components/layouts/AccountLayout/AccountPageContainer';
+import { getAccountLayout } from '../../components/layouts/AccountLayout';
+import { defaultSeo } from '../../next-seo';
+import { getTemplatedTitle } from '../../components/layouts/utils';
+
+const seo: NextSeoProps = {
+  ...defaultSeo,
+  title: getTemplatedTitle('Manage plus'),
+};
+
+const AccountManageSubscriptionPage = (): ReactElement => {
+  const router = useRouter();
+  const { isPlus, plusProvider } = usePlusSubscription();
+
+  useEffect(() => {
+    if (!isPlus) {
+      router.push(plusUrl);
+      return;
+    }
+
+    if (plusProvider === SubscriptionProvider.Paddle) {
+      router.push(managePlusUrl);
+      return;
+    }
+
+    if (isIOSNative() && plusProvider === SubscriptionProvider.AppleStoreKit) {
+      globalThis.webkit.messageHandlers[
+        WebKitMessageHandlers.IAPSubscriptionManage
+      ].postMessage(null);
+
+      // Send the user back to the previous page as the native handler will take over
+      router.back();
+    }
+  }, [isPlus, plusProvider, router]);
+
+  if (
+    !isPlus ||
+    plusProvider === SubscriptionProvider.Paddle ||
+    (isIOSNative() && plusProvider === SubscriptionProvider.AppleStoreKit)
```

Comment:
What? 😆

Reply:
Same statements as the useEffect above, just to make sure we don't render anything on the page while router is doing its work.

Reply:
Probably, going to test it